perceval/backends/mozilla_reps.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#     Alvaro del Castillo <acs@bitergia.com>
+#
+
+import json
+import logging
+import os.path
+
+import requests
+
+from dateutil import parser
+
+from ..backend import Backend, BackendCommand, metadata
+from ..cache import Cache
+from ..errors import BackendError, CacheError, ParseError
+
+from ..utils import (DEFAULT_DATETIME,
+                     datetime_to_utc,
+                     str_to_datetime,
+                     urljoin)
+
+logger = logging.getLogger(__name__)
+
+class MozillaReps(Backend):
+    """MozillaReps backend for Perceval.
+
+    This class retrieves the events from a
+    MozillaReps url. To initialize this class a
+    url could be provided. If not, https://reps.mozilla.org will be used.
+
+    :param url: MozillaReps url
+    :param cache: cache object to store raw data
+    :param origin: identifier of the repository; when `None` or an
+        empty string are given, it will be set to `url` value
+    """
+    version = '0.1.0'
+
+    def __init__(self, url=None, cache=None, origin=None):
+        if not url:
+            url = "https://reps.mozilla.orghttps://reps.mozilla.org "
```

Comment:
This should be a constant outside this class. In fact, there's a bug on the URL.

URL field should be a mandatory argument, I think.